plugins/chat/app/services/chat/create_message_interaction.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Chat
+  # Service responsible for creating and validating a new interaction between a user and a message.
+  #
+  # @example
+  #  Chat::CreateMessageInteraction.call(params: { message_id: 3, action_id: "xxx" }, guardian: guardian)
+  #
+  class CreateMessageInteraction
+    include ::Service::Base
+
+    # @!method self.call(guardian:, params:)
+    #   @param [Guardian] guardian
+    #   @param [Hash] params
+    #   @option params [Integer] :message_id
+    #   @option params [Integer] :action_id
+    #   @return [Service::Base::Context]
+    params do
+      attribute :message_id, :integer
+      validates :message_id, presence: true
```

Comment:
Could we group the validations together instead?

Reply:
I will make the change, but don't really agree. For me similar concerns and the most likely to change together should be grouped.